machine/delta/delta_tn48m/kernel/series

```diff
@@ -0,0 +1,2 @@
+0001-Marvell-support-for-armada7020.patch
```

Comment:
The board has a Armada 7040, and the patch seems to add much more than what is needed, which makes it hard to review, and to upstream.

What is missing in Linux upstream before the patch can be dropped?

@kostapr do you have more information?